CMS-Enterprise/easi-app
# [NOREF] Upgrade sass, cypress, and build tooling

# NOREF

## Description

- Upgrades `sass` and moves it to devDependencies
- Upgrades several other build, cypress, and random tooling packages from https://github.com/CMS-Enterprise/easi-app/pull/2829

## How to test this change

- Make sure that tests pass in E2E / Cypress
- Make sure that `sass` upgrade didn't unexpectedly modify any styles

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
